pyiron/atomistics/structure/atoms.py

```diff
@@ -552,10 +555,10 @@ def from_hdf(self, hdf, group_name="structure"):
                 self.dimension = hdf_atoms["dimension"]
                 self.units = hdf_atoms["units"]
 
-                self.cell = None
+                self._cell = None
                 if "cell" in hdf_atoms.list_groups():
                     with hdf_atoms.open("cell") as hdf_cell:
-                        self.cell = hdf_cell["cell"]
+                        self._cell = hdf_cell["cell"]
```

Comment:
I guess for consistency we should always use `set_cell` instead of `self._cell`.